src/infrastructure/repositories/figma-team-repository.integration.test.ts

```diff
@@ -60,42 +60,21 @@ describe('FigmaTeamRepository', () => {
 					),
 				]);
 			const [targetFigmaTeam1, targetFigmaTeam2] = await Promise.all([
-				figmaTeamRepository
```

Comment:
**note:** Remov unnecessary code which was accidentally discovered while working in integration tests. Likely, this code is the result of the incorrect recent merge.